posts/2020-11-27-Rustup-1.23.0.md

```diff
@@ -0,0 +1,88 @@
+---
+layout: post
+title: "Announcing Rustup 1.23.0"
+author: The Rustup Working Group
+---
+
+The rustup working group is happy to announce the release of rustup version 1.23.0. [Rustup][install] is the recommended tool to install [Rust][rust], a programming language that is empowering everyone to build reliable and efficient software.
+
+If you have a previous version of rustup installed, getting rustup 1.23.0 is as easy as closing your IDE and running:
+
+```
+rustup self update
+```
+
+Rustup will also automatically update itself at the end of a normal toolchain update:
+
+```
+rustup update
+```
+
+If you don't have it already, you can [get rustup][install] from the appropriate page on our website.
+
+[rust]: https://www.rust-lang.org
+[install]: https://rustup.rs
+
+## What's new in rustup 1.23.0
+
+This release has a few important things we'd like to hilight:
+
+1. `rustup` is now available for the Apple M1 based computers. We now have an
+   `aarch64-apple-darwin` release of `rustup` which means that you ought to
+   be able to install a native `rustup` and compiler toolset on that platform.
+   **NOTE:** As of the time of this blog post, there is no _stable_ release of
+   the M1 toolchain, so you will need to select `beta` or `nightly` when
+   prompted for a toolchain to install.
```

Comment:
We should mention that support for the target is Tier 2, and link to https://github.com/rust-lang/rust/issues/73908 to follow the progress on that target.